src/applications/caregivers/components/SubmitError/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+const SubmitError = () => {
+  const ErrorBody = () => {
+    return (
+      <div>
+        <p>
+          We’re sorry. Something went wrong when you tried to submit your
+          application. Please review your application to make sure you entered
+          your information correctly. If you think your information is correct,
+          please call the VA.gov help desk at 855-574-7286 (TTY: 711). We’re
+          here Monday through Friday, 8:00 a.m.–8:00 p.m. ET.
```

Comment:
For accessibility reasons, please don't use a dash: `8:00 a.m to 8:00 p.m. ET.` ([ref](https://design.va.gov/content-style-guide/dates-and-numbers#times-and-time-zones)).

Reply:
Roger that just copied and pasted designs. Will let @jonathanjnelson know as well